plugins/micromega/dune

```diff
@@ -1,18 +1,26 @@
+(library
+ (name micromega_core_plugin)
+ (public_name coq-core.plugins.micromega-core)
```

Comment:
It looks like it doesn't like the dash in `micromega-core`. Maybe it needs `(name ${name}_plugin)` and `(public_name coq-core.plugins.${name})` to use the same basic ${name} somehow?